pandas/io/feather_format.py

```diff
@@ -93,6 +93,11 @@ def read_feather(path, use_threads=True):
     Parameters
     ----------
     path : string file path, or file-like object
+    columns : sequence, default None
+        Only read a specific set of columns. If not provided, all columns are
+        read
+
+        .. versionadded 0.24.0
```

Comment:
* I think we generally prefer say array-like instead of sequence.
* Nit: can we somehow remove the dangling word on the second line of your description?

Reply:
array-like implies a `.dtype`. This just needs to be a sequence.

Reply:
> * Nit: can we somehow remove the dangling word on the second line of your description?

```
If not provided, all columns are read
```
should do?

Reply:
Yep, that should work!